### DISCUSSION TITLE
Set custom Remote-User header

### DISCUSSION THREAD

**Discussion by zkvvoob** [goauthentik/authentik#5273] [Q&A]
Hi all!

I'm trying to set up FreshRSS, which seems to only accept the Remote-User header - and not HTTP_REMOTE_USER, for some reason. I tried passing the following directive to Nginx Proxy Manager: `proxy_set_header Remote-User $authentik_username`, but that seems to produce `HTTP_REMOTE_USER`, which I have no use of.

I've also set a custom Scope mapping like this
```python
return {
    "ak_proxy": {
        "user_attributes": {
            "additionalHeaders": {
                "Remote-User": request.user.username
            }
        }
    }
}
```
But I can't see the header anywhere, even though the Scope mapping *is* added to the Proxy Provider. 

Could you help me figure out what I'm doing wrong?

**Comment by moutasem1989** [comment]
Have you managed to figure out how to call `additionalHeaders`?
I am trying to understand this on NginX but there is no clear documentation and im a visual learner
